framework/CodeInterfaces/Saphire/SaphireInterface.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on July 12, 2018
+
+@author: wangc
+
+comments: Interface for SAPHIRE Simulation
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+
+import os
+import copy
+
+from GenericCodeInterface import GenericCode
+from SaphireData import SaphireData
+
+class Saphire(GenericCode):
+  """
+    SAPHIRE Interface.
+  """
+  def __init__(self):
+    """
+      Initializes the SAPHIRE code interface
+      @ In, None
+      @ Out, None
+    """
+    GenericCode.__init__(self)
+    self.codeOutputs = {} # the root of the outputs
```

Comment:
is a dict...so I imagine there are multiple roots. Please better explanation here

Reply:
added.